### ISSUE TITLE
Make Zubat theme tabs more readable

### ISSUE THREAD

**Issue by nblackburn** [klauscfhq/hyper-pokemon#30]
![Zubat](https://user-images.githubusercontent.com/2931085/36674726-c7fa952c-1afe-11e8-8ce6-c104897d04a9.png)

The Zubat theme tabs need tweaking to be more readable as it's hard to make them out.

https://github.com/klauscfhq/hyper-pokemon/blob/master/pokemon.yml#L292-L297

**Comment by klaudiosinani** [comment]
It is possible to manually change the tabs color through the `pokemonSyntax` option, by setting it to `dark`, thus achieving a better contrast. This is something that will happen automatically on the upcoming release.

```js
// ~/.hyper.js
pokemon: 'zubat',
pokemonSyntax: 'dark', // Defines the color of the terminal tabs    
```
![hyper_2018-03-23_22-40-03](https://user-images.githubusercontent.com/12670537/37852410-2463d8a0-2eeb-11e8-8659-4bc8caeb1333.png)

**Comment by klaudiosinani** [comment]
Fixed on [`19311b5`](https://github.com/klauscfhq/hyper-pokemon/commit/19311b57320955e4e96fb17bff56564b33135d05).
